build-tools/build-release-images.sh

```diff
@@ -20,12 +20,11 @@ VERSION_INFO=$(${CURDIR}/version-tool version)
 # Hard code the platform dir here
 cp $CURDIR/../_docker_workspace/out/$RELEASE_PLATFORM/bin/* $WKDIR/
 mkdir -p $WKDIR/python
-cp python/*.py $WKDIR/python/
-cp python/cf-runtime-requirements.txt $WKDIR/
+cp requirements.txt $WKDIR/
 cp schema/cf-schema_v*.json $WKDIR/
 
 echo "{\"version\": \"${VERSION_INFO}\", \"build\": \"${BUILD_INFO}\"}" \
-  > $WKDIR/python/VERSION_BUILD.json
+  > $WKDIR/VERSION_BUILD.json
```

Comment:
Same comment here about the VERSION_BUILD not being needed.

Reply:
Sam comment on why its needed lol.